src/dash-table/utils/applyClipboardToData.ts

```diff
@@ -56,8 +58,8 @@ export default (
     const lastEntry = derived_viewport_indices.slice(-1)[0] || 0;
     const viewportSize = derived_viewport_indices.length;
 
-    values.forEach((row: string[], i: number) =>
-        row.forEach((cell: string, j: number) => {
+    for (let [i, row] of values.entries()) {
+        for (let [j, value] of row.entries()) {
```

Comment:
using for-of instead of forEach makes it possible to terminate the loop execution (return in fn scope instead of lambda)